.github/workflows/test-dev.yml

```
@@ -0,0 +1,55 @@
+      - name: 실행권한 부여
+        run: chmod +x gradlew
+
+      - name: 테스트
+        run: ./gradlew test
+
+      - name: 테스트 결과 리포트 생성
+        uses: EnricoMi/publish-unit-test-result-action@v2
+        if: always()
+        with:
+          files: build/test-results/test/*.xml
+          if-no-files-found: warn
```

actionlint (1.7.12)

[error] input "if-no-files-found" is not defined in action "EnricoMi/publish-unit-test-result-action@v2". available inputs are "action_fail", "action_fail_on_inconclusive", "check_name", "check_run", "check_run_annotations", "check_run_annotations_branch", "comment_mode", "comment_title", "commit", "compare_to_earlier_commit", "deduplicate_classes_by_file_name", "event_file", "event_name", "fail_on", "files", "github_retries", "github_token", "github_token_actor", "ignore_runs", "job_summary", "json_file", "json_suite_details", "json_test_case_results", "json_thousands_separator", "junit_files", "large_files", "nunit_files", "pull_request_build", "report_individual_runs", "report_suite_logs", "search_pull_requests", "secondary_rate_limit_wait_seconds", "seconds_between_github_reads", "seconds_between_github_writes", "ssl_verify", "test_changes_limit", "test_file_prefix", "time_unit", "trx_files", "xunit_files"

(action)